addons/test_mail/tests/test_mail_activity.py

```diff
@@ -218,6 +218,22 @@ def test_action_feedback_attachment(self):
 @tests.tagged('mail_activity')
 class TestActivityMixin(TestActivityCommon):
 
+    @classmethod
+    def setUpClass(cls):
+        super(TestActivityMixin, cls).setUpClass()
+
+        cls.user_utc = cls.env['res.users'].create({
```

Comment:
Could probably use mail_new_test_user helper (sets context to avoid password template, ...)